src/i_oalsound.c

```diff
@@ -0,0 +1,660 @@
+//
+// Copyright(C) 2023 Roman Fomin
+// Copyright(C) 2023 ceski
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// DESCRIPTION:
+//      System interface for OpenAL sound.
+//
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "doomstat.h"
+#include "i_oalsound.h"
+#include "r_data.h"
+
+#define OAL_ROLLOFF_FACTOR 1
+#define OAL_SPEED_OF_SOUND 343.3f
+#define OAL_MAP_UNITS_PER_METER (128.0f / 3.0f) // 128 map units per 3 meters.
```

Comment:
Is there a reference for that? Doomwiki?

Reply:
Yes, please.

Reply:
I saw that section but it's not consistent with the table. There are many ways to interpret the dimensions, so I just picked one.

Added a citation here: https://github.com/fabiangreffrath/woof/pull/1118/commits/513a60c4c451b8254bc9979560906ab534d1bef6